lib/prawn-rails-forms.rb

```diff
@@ -6,6 +6,13 @@
 require 'prawn-rails'
 
 module PrawnRailsForms
+  def default_text_field_options=(options)
+    @options = options
+  end
+
+  def default_text_field_options
+    @options || {}
```

Comment:
I think this should be `||=`. That way, I think this would work:

```ruby
PrawnRailsForms.default_text_field_options[:crunchy] = true
```

even when there aren't any options defined yet.

Reply:
I'm confused. Wouldn't I need to define the `[]=` method for that to work?

Reply:
I don't think so. `#default_text_field_options` returns a `Hash` object, which already has `#[]` and `#[]=` defined on it. But, you always want to return the _particular_ hash that is stored in the instance variable, which is why I suggest `||=`

Reply:
```
[5] pry(#<#<Class:0x00000003124aa0>>)> PrawnRailsForms.default_text_field_options={style: :italic}
=> {:style=>:italic}
[6] pry(#<#<Class:0x00000003124aa0>>)> PrawnRailsForms.default_text_field_options[:thing]
=> nil
[7] pry(#<#<Class:0x00000003124aa0>>)> PrawnRailsForms.default_text_field_options[:thing] = 3
=> 3
[8] pry(#<#<Class:0x00000003124aa0>>)> PrawnRailsForms.default_text_field_options[:thing]
=> 3
[9] pry(#<#<Class:0x00000003124aa0>>)> PrawnRailsForms.default_text_field_options
=> {:style=>:italic, :thing=>3}
```

Reply:
Gotcha. I understand now.